adserver/tasks.py

```diff
@@ -841,6 +841,71 @@ def notify_of_publisher_changes(difference_threshold=0.25, min_views=10_000):
                     )
 
 
+@app.task()
+def disable_inactive_publishers(days=60, draft_only=False, dry_run=False):
+    """Disable publishers who haven't had a paid impression in the specified `days`."""
+    if days < 1:
+        # Prevent the misstep where days is <=0 and all publishers are marked inactive
+        return
```

Comment:
Seems like we probably want to set this to `<30`? And also log a warning instead of failing silently.